src/frontend/src/components/AuthorResult/AuthorResult.jsx

```diff
@@ -1,53 +1,68 @@
-import CreateIcon from '@material-ui/icons/Create';
 import EventIcon from '@material-ui/icons/Event';
 import PermContactCalendarIcon from '@material-ui/icons/PermContactCalendar';
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
-import Paper from '@material-ui/core/Paper';
-import Grid from '@material-ui/core/Grid';
 import Avatar from '@material-ui/core/Avatar';
-import List from '@material-ui/core/List';
-import ListItem from '@material-ui/core/ListItem';
-import ListItemIcon from '@material-ui/core/ListItemIcon';
-import ListItemText from '@material-ui/core/ListItemText';
+import Container from '@material-ui/core/Container';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
```

Comment:
Small nit and I can slam on the Approve button. You can combine these imports into one line
`import { Avatar, Card, CardActionArea, CardContent, CardMedia, Container, Typography } from '@material-ui/core';`

Reply:
Resolved.